README.md

```diff
@@ -5,11 +5,14 @@
 This module provides support for bean validation 2.0 (JSR380). Can be used with any service provider of the bean validation spec 
 e.g. `org.hibernate.validator:hibernate-validator`
 
-Added validations:
+Features:
 
 - `@Size` for vavr's `Traversable<T>`
 - `@NotEmpty` for vavr's `Value<T>`
-- All available validations can be applied to nested Tuple Values. see example below
+- All available validations can be applied to nested Tuple Values. See example below
+- All available validations can be applied to vavr's `Map<K, V>`s and `Traversable<T>`s
```

Comment:
I've added `Traversable` here. Is it supported? Or it should be `Seq` here?

Reply:
Good question, I thought it was self explanatory but it is not. I will think about it.
Essentially it works on all vavr types since the inherit from `Iterable`.

I think I will add some more examples